Bottom: 538d595208168a57edcc59b9a6cc30eb8fd30afc
Top:    0f75dfa919ae570d1dc281158f3363038babef3c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 15:39:34 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
